app/helpers/submissions_helper.rb

```diff
@@ -246,6 +246,10 @@ def render_submission_inputs(frame_id)
     end
 
     submission_metadata.reject { |attr| reject_metadata.include?(attr['attribute']) || !selected_attribute?(attr['attribute']) }.each do |attr|
+      attr['label'] = "Modified on" if attr['attribute'] == "modificationDate"
```

Comment:
I will remove this, as it will be fixed in the back. But you did a good job understanding that this code block rendered the inputs.   

@jonquet, concerning these new labels that you set in the Google sheet. They are not the same as the ones that you set in the YAML file, for example `released` attribute, is `Submission date` but you asked in the Google sheet to put it to `Initially created on`. In all the cases this will handled in the backend.